.github/actions/cross-build-binary/action.yml

```diff
@@ -56,12 +56,23 @@ runs:
         name: ${{ env.ASSET_FULL_NAME }}.tar.gz
         path: ./${{ env.ASSET_FULL_NAME }}.tar.gz
         retention-days: 3
-    - name: Upload archive
+    - name: Deploy archive to GitHub nightly release
+      if: "${{ inputs.version == 'nightly' }}"
+      uses: quickwit-inc/upload-to-github-release@v1
+      env:
+        GITHUB_TOKEN: ${{ inputs.token }}
+      with:
+        file: ${{ env.ASSET_FULL_NAME }}.tar.gz
+        overwrite: true
+        draft: false
```

Comment:
does `draft: ${{ inputs.version == 'nightly' }}` can work?

Reply:
I don't think so but let me try. I am not a fan of the verbose solution so I am still trying to find a better way.

Reply:
Thanks @fmassot 🎖️  the expression `draft: ${{ inputs.version != 'nightly' }}` works so no need for a huge duplicate.